CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+  	picture = "gelli_andreotti"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1973
+		NOT = { year = 1992 }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = conservative
+		
+		has_country_modifier = dirty_war
+		
+		has_country_flag = ITA_P2
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 6 }
+	
+	immediate = { set_country_flag = ITA_P2_2 }
+	
+	option = {
+		name = "Enact the reforms"
+		prestige = 25
+		
+		money = 100000
+		
+		any_pop = {  
+		    limit = { strata = poor }
+			consciousness = 4
+			militancy = 2
+		}
+		
+		any_country = {
+			limit = { has_country_modifier = alliance_nato } 
+			relation = { who = THIS value = 25 } 
+		}
+		
+		add_country_modifier = { name = economic_boom duration = 365 }
+		add_country_modifier = { name = bilateral_trade_pact_gp duration = 365 }
+		add_country_modifier = { name = local_stock_exchange duration = 365 }
+		ai_chance = { factor = 0.9 }
+	}
+
+	option = {
+		name = "Don't enact the reforms"
+		prestige = -25
+		
+		any_pop = {  
+		    limit = { strata = rich }
+			consciousness = 4
+			militancy = 2
+		}
+ 
+        set_country_flag = ITA_P2_Econ_fail
+		ai_chance = { factor = 0.1 }
+	}
+
+}
+
+#Democratic Reorganization - Mafia
+country_event = {
+  	id = 8020420
+  	title = "The Status of our Associates"
+  	desc = EVT_8020420_DESC
+  	picture = "italian_parliament"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1973
+		NOT = { year = 1992 }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = conservative
+		
+		has_country_modifier = dirty_war
+		
+		has_country_flag = ITA_P2
+		has_country_flag = ITA_P2_2
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 6 }
+	
+	immediate = { clr_country_flag = ITA_P2_2 set_country_flag = ITA_P2_3 }
+	
+	option = {
+		name = "Accept the deal"
+		prestige = -10
+		
+		money = -200000
+		
+		any_pop = {  
+		    limit = { OR = { strata = rich strata = middle } location = { OR = { region = PAP_752 region = SIC_753 region = SIC_754 region = SIC_760 region = SIC_763 region = SAR_765 } } }
+			money = 100
+			consciousness = -9
+			militancy = -9
+		}
+		any_pop = { 
+			limit = { OR = { type = bureaucrats type = officers type = soldiers } }
+			money = 50
+		}
+		
+		ai_chance = { factor = 0.9 }
+	}
+
+	option = {
+		name = "Sideline them"
+		prestige = 10
+		
+		money = -100000
+		
+		small_arms = -2000
+	    weaponry = -2000
+	    ships = -200
+	    aeroplanes = -200
+		
+		any_pop = {  
+		    limit = { OR = { strata = rich strata = middle } location = { OR = { region = PAP_752 region = SIC_753 region = SIC_754 region = SIC_760 region = SIC_763 region = SAR_765 } } }
+			money = -100
+			consciousness = 9
+			militancy = 9
+		}
+		any_pop = { 
+			limit = { OR = { type = bureaucrats type = officers type = soldiers } }
+			money = 100
```

Comment:
This does not work. Please delete this any_pop bracket and use
bureaucrats = { money = 50 }
officers = { money = 50 }
soldiers = { money = 50 }